docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -25,6 +25,13 @@ cluster in a single JSON file.
 Configuration
 -------------
 
+.. warning::
```

Comment:
I think this should be part of the release notes. If we add warnings like that we may end up with messy docs.

Reply:
Agreed, and it should be in a breaking change section, as this will break expected security behaviors.

Reply:
I was told though that there are no special sections, and release notes are generated from issues?

Reply:
Release notes are not generated. We write them .. but we do not have a dedicated breaking changes section. But this can be added .. probably was already .. @colebow ?

Also the warning is still appropriate .. maybe without the version info and past behavior though

Reply:
I remove information about Trino versions

Reply:
I think we still need this too in release notes.